architecture/core_objects/routing.adoc

```diff
@@ -299,30 +307,32 @@ link:#special-notes[special notes] below.
 [[wildcard-certificates]]
 *Wildcard Certificates*
 
-Based on the implementation, you may be able to use a default certificate. Default certificates
-are useful for implementing a wildcard certificate for the router.  For example, if you have
-many routes that end in example.com you may wish to install a router with a wild card
-certificate for `*.example.com`.
+Based on the implementation, you can use a default certificate. Default
+certificates are useful for implementing a wildcard certificate for the router.
+For example, if you have many routes that end in example.com you may wish to
+install a router with a wildcard certificate for `*.example.com`.
 
```

Comment:
Maybe "Depending on implementation..."  ?
Also probably should mark up first example.com for consistency w/ second.